packages/tailwindcss/src/utilities.ts

```diff
@@ -1282,6 +1309,27 @@ export function createUtilities(theme: Theme) {
     },
   ])
 
+  for (let axis of ['x', 'y']) {
+    functionalUtility(`rotate-${axis}`, {
+      supportsNegative: true,
+      supportsFractions: true,
```

Comment:
I don't think we are doing anything with the fraction, so I think we can drop this?